yaksh/templates/yaksh/grade_user.html

```diff
@@ -342,19 +251,58 @@ <h5>No Assignment submitted by {{ data.user.get_full_name.title }}</h5>
         {% endfor %}
       </div>
         </div>
-
         {% else %}
-         <div class="card">
-            <div class="card-body">
-          {{ ans.answer.answer.strip|safe }}
-          </div>
-        </div>
         {% endif %}
-        </div>
-        </div>
         {% endfor %}
     {% endif %}
     {% with answers|last as answer %}
+    <div class="card">
+          <div class="card-body">
+          {% if question.type == "code" %}  
+              {% for ele in diff %}
+                {% for key, value in ele.items %}
+                  {% if key == question.summary %}
+                    <font color="red" size="4"><b>{{question.summary}}</b></font>
+                    </br>
+                    <pre><code>{{value.0}}</code></pre>
+                    <hr style="border-top: 3px solid #ccc; background: transparent;"> 
+                    </br>
+                    {% for tuple_of_four in value.1 %}
```

Comment:
Same rules of variable naming apply here as well, what is tuple_of_4 and what is value or value.1? Please name variables keeping in mind that the intent of the information should be clear.